source/develop/release-management/features/storage/incremental-backup.html.md

```diff
@@ -199,47 +205,47 @@ Response:
 <disks>
     <disk>
         ...
-        <backup>incremental|null</backup>
+        <backup>incremental|none</backup>
         ...
-    </disks>
+    </disk>
     ...
 </disks>
 ```
 
 #### Starting backup
 
-Start incremental backup since backup id "previous-backup-uuid".
+Start incremental backup since checkpoint id "previous-checkpoint-uuid".
 
 Request:
 ```
 POST /vms/vm-uuid/backups
 
-<backup>
-    <incremental>previous-backup-uuid</incremental>
+<vm_backup>
+    <checkpoint>previous-checkpoint-uuid</checkpoint>
     <disks>
         <disk id="disk-uuid" />
         ...
     </disks>
-</backup>
+</vm_backup>
 ```
 
 Response:
 ```
-<backup id="backup-uuid">
-    <incremental>previous-backup-uuid</incremental>
+<vm_backup id="backup-uuid">
+    <checkpoint>previous-checkpoint-uuid</checkpoint>
     <disks>
         <disk id="disk-uuid" />
         ...
         ...
     </disks>
-    <status>initiailizing</status>
-    <creation_date>...
-</backup>
+    <phase>initiailizing</phase>
```

Comment:
Don't we use ```status``` in other places?

"Wait until backup status is ready" sounds better than "Wait until backup phase is ready"